app/controllers/users/reset_passwords_controller.rb

```diff
@@ -22,6 +22,7 @@ def edit
       result = PasswordResetTokenValidator.new(token_user).submit
 
       analytics.password_reset_token(**result.to_h)
+      irs_tracker_forgot_password_email_confirmed(result)
```

Comment:
same as before, I don't think this needs a separate method, it's simple enough to inline IMO

```suggestion
      irs_attempts_api_tracker.forgot_password_email_confirmed(
        success: result.success?,
        failure_reason: result.errors,
      )
```